### PR TITLE
Intly 3122 update route

### PR DESCRIPTION
**Pull request by austinmartinh**
# Description
JIRA: https://issues.redhat.com/browse/INTLY-3122

### CODE REVIEW

File: manifests/integreatly-solution-explorer/solution-explorer-0.0.48/solution-explorer-operator.v0.0.48.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                  - command:
+                      - tutorial-web-app-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: OPERATOR_NAME
+                        value: tutorial-web-app-operator
+                    image: quay.io/dimitra/tutorial-web-app-operator:v0.0.48
```
Comment (dimitraz) [inline manifests/integreatly-solution-explorer/solution-explorer-0.0.48/solution-explorer-operator.v0.0.48.clusterserviceversion.yaml:44]: Leaving my org in here until https://github.com/integr8ly/tutorial-web-app-operator/pull/89 is merged

**Comment by grdryn** [comment]
I've verified this. No point in adding labels yet though, since they'll just get removed when it changes to remove &imitra's custom image.

**Comment by dimitraz** [comment]
&imitra is a new one, I like it

**Pull request merged**